libs/entitlement/qa/entitlement-test-plugin/src/main/java/org/elasticsearch/entitlement/qa/test/PathActions.java

```diff
@@ -37,7 +38,8 @@ static void checkToRealPathForInvalidTarget() throws IOException {
         try {
             EntitledActions.pathToRealPath(invalidLink); // throws NoSuchFileException when checking entitlements due to invalid target
         } catch (NoSuchFileException e) {
-            assert Arrays.stream(e.getStackTrace()).anyMatch(t -> t.getClassName().equals(PolicyManager.class.getName()))
+            assert Arrays.stream(e.getStackTrace())
+                .anyMatch(t -> Objects.equals(t.getModuleName(), PolicyChecker.class.getModule().getName()))
```

Comment:
Why did this change?

Reply:
It had to change to `PolicyCheckerImpl.class` anyway, and it felt a little icky to hardwire this to point at an implementation class, so I figure there should be a frame that's inside the entitlement library's main package.

I don't feel strongly; I can put it back to testing `PolicyCheckerImpl.class` if you prefer that.